src/main/java/net/quickwrite/fluent4j/parser/FluentParser.java

```diff
@@ -17,7 +17,7 @@ public class FluentParser {
     private int index = 0;
 
     public FluentParser(String input) {
-        this.input = input;
+        this.input = input.replace("\r", "");
```

Comment:
This is more or less a lousy fix and should be done better inside of the `StringSlice`.

Reply:
I think I will change this in a later PR.